pkg/pillar/cmd/verifier/cmd/oneshot.go

```diff
@@ -0,0 +1,64 @@
+package main
```

Comment:
Please add Copyright and License lines to all new files.

Reply:
Oops. Yeah, adding now.

Reply:
Done. Thank you for reminding me.